network-api/networkapi/wagtailpages/customblocks/__init__.py

```diff
@@ -0,0 +1,73 @@
+from wagtail.images.blocks import ImageChooserBlock
+from wagtail.core import blocks
+
+from .air_table_block import AirTableBlock
+from .bootstrap_spacer import BootstrapSpacerBlock
+from .iframe_block import iFrameBlock
+from .annotated_image_block import AnnotatedImageBlock
+from .image_grid import ImageGrid, ImageGridBlock
+from .image_text_mini import ImageTextMini, ImageBlock
+from .image_text import ImageTextBlock
+from .latest_profile_list import LatestProfileQueryValue, LatestProfileList
+from .link_button import LinkButtonBlock
+from .profile_by_id import ProfileById
+from .profile_directory import ProfileDirectory
+from .pulse_project_list import PulseProjectList
+from .quote_block import QuoteBlock
+from .video_block import VideoBlock
+
+__all__ = [
+    'AirTableBlock',
+    'BootstrapSpacerBlock',
+    'iFrameBlock',
+    'AnnotatedImageBlock',
+    'ImageGrid',
+    'ImageGridBlock',
+    'ImageTextMini',
+    'ImageTextBlock',
+    'LatestProfileQueryValue',
+    'LatestProfileList',
+    'LinkButtonBlock',
+    'ProfileById',
+    'ProfileDirectory',
+    'PulseProjectList',
+    'QuoteBlock',
+    'VideoBlock'
+]
+
+# Deprecated/ Unused ?
+
+
+class FigureBlock2(blocks.StructBlock):
```

Comment:
Filed a [follow-up ticket](https://github.com/mozilla/foundation.mozilla.org/issues/3003) to remove this.